platform/broadcom/sonic-platform-modules-tencent/common/lib/algorithm/hysteresis.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
```

Comment:
not sure why we need to declare since UTF-8 is the default source encoding:-[https://peps.python.org/pep-3120/](https://peps.python.org/pep-3120/)

Reply:
This is redundant code and has been removed，thandks